crates/state/src/bridge_state.rs

```diff
@@ -94,6 +94,7 @@ impl OperatorTable {
     }
 
     /// Sanity checks the operator table for sensibility.
+    #[allow(dead_code)] // #FIXME: remove this.
```

Comment:
Moving forward we should use `#[expect(dead_code)]` instead. And then, we don't need to rely on `FIXME:`. The compiler will tell us if this can be removed.